src/web/mod.rs

```diff
@@ -496,6 +496,28 @@ fn redirect(url: Url) -> Response {
     resp
 }
 
+/// creates a redirect-response which is cached on the CDN level for
+/// the given amount of seconds. Browser-Local caching is deactivated.
+/// The used s-maxage is respected by CloudFront (which we can invalidate
+/// if we need to), and perhaps by other proxies in between.
+/// When the cache-seconds are reasonably small, this should be fine.
+///
+/// CloudFront doesn't have `surrogate-control` headers which would be only
+/// used by CloudFront and dropped (Fastly has them).
+///
+/// The `Expires` header from `redirect` is still kept to be safe,
```

Comment:
What sets 'Expires'? How is it determined?

Reply:
@jyn514 it's set in the method above ( `fn redirect`),  

I found every other place in this codebase to add the (old) `Expires` header for redirects, going back to [an initial commit 5 years ago](https://github.com/rust-lang/docs.rs/commit/f59bc5f030ef9d5e220bd6503c9e5fa5efbb6264#diff-e7bb401ce4e1deafed2f3826e6fa900e3f76181906759cbbec3eaf92812381d9R78). 

I'm not sure why this was added, to my knowledge setting `Cache-Control` with `max-age` or `s-maxage` is enough for all proxies and browser, but who knows, the web is a world of old software and edge-cases :)

Reply:
Cache-Control is [supported on all browsers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#browser_compatibility) (even weird ones like Samsung Internet) so I think we can get rid of `Expires`.

Reply:
@jyn514 kk, I'll do that.